### PR TITLE
[disney] provide episode_number and season_number for some series

### PR DESCRIPTION
**Pull request by saper**
For #13299.

Not yet tested

### CODE REVIEW

**Comment by dstftw** [comment]
Read https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions and fix all issues.

File: youtube_dl/extractor/disney.py [hunk 1]
```diff
@@ -160,3 +162,10 @@ def _real_extract(self, url):
             'formats': formats,
             'subtitles': subtitles,
         }
+
+        if len(externals) > 0:
+            if 'data' in externals[0]:
+                metadata['season_number'] = int_or_none(externals[0]['data'].get('Season', None))
+                metadata['episode_number'] = int_or_none(externals[0]['data'].get('Episode', None))
```
Comment (saper) [inline youtube_dl/extractor/disney.py]: @dstftw do you mean I should use `externals[0].get('data', {})` here?

Comment (dstftw) [inline youtube_dl/extractor/disney.py]: No. `utils.try_get`.

**Comment by keybounce** [comment]
OK, something went wrong for me on this one.

```
keybounceMBP:Breakthrough michael$ ./yt-dl -v -a dl-list 
Traceback (most recent call last):
  File "/Users/michael/bin/youtube-dl/youtube_dl/extractor/__init__.py", line 4, in <module>
ModuleNotFoundError: No module named 'youtube_dl.extractor.lazy_extractors'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/michael/bin/youtube-dl/__main__.py", line 16, in <module>
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/Users/michael/bin/youtube-dl/youtube_dl/__init__.py", line 43, in <module>
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/Users/michael/bin/youtube-dl/youtube_dl/extractor/__init__.py", line 9, in <module>
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "/Users/michael/bin/youtube-dl/youtube_dl/extractor/extractors.py", line 293, in <module>
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 946, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 885, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1157, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1131, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1112, in _legacy_get_spec
  File "<frozen importlib._bootstrap>", line 427, in spec_from_loader
  File "<frozen importlib._bootstrap_external>", line 544, in spec_from_file_location
  File "/Users/michael/bin/youtube-dl/youtube_dl/extractor/disney.py", line 166
    data_pack = externals[0].get('data', {})
                                           ^
TabError: inconsistent use of tabs and spaces in indentation
keybounceMBP:Breakthrough michael$ 
```

```
keybounceMBP:youtube-dl michael$ git status
On branch newJuly
nothing to commit, working tree clean
keybounceMBP:youtube-dl michael$ git log --author=keybounce --pretty=oneline 
43f2a4c2098bad109e704c3ae3248437f77cf36e (HEAD -> newJuly) Merge remote-tracking branch 'origin/pr/13384' into newJuly
d4b835327a9ae93ad35733f452b883ac2e2af880 Merge branch 'CBSPlayList' into newJuly
90139762e5cd07d774fa43cffabcf356b7cce5a6 Merge branch 'Dish' into newJuly
keybounceMBP:youtube-dl michael$ 
```

My youtube-dl has just the DishMSO, the CBS playlist, and this. 

I have no clue what's going on, except for the inconsistent space/tab issue. I could probably fix that, but the lazy extractor issue I have no clue about. My previous youtube-dl mix did not have this problem (with the Dish, CBS playlist, the 4 lines for disney that gkoelln gave, the previous disney/marvel enhanced support that he had, and an earlier master ... current master gave a conflict so I started over.)

**Pull request closed without merging**